<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addProfile</class>
 <widget class="QDialog" name="addProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="profileLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>15</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 26pt &quot;Assassin$&quot;;</string>
   </property>
   <property name="text">
    <string>Please enter profile name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="profileNameLineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>571</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 26pt &quot;Assassin$&quot;;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="profileModNumberSpinBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>120</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
font: 26pt &quot;Assassin$&quot;;</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="profileNumberLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>581</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 26pt &quot;Assassin$&quot;;</string>
   </property>
   <property name="text">
    <string>Please enter amount of mods in the profile</string>
   </property>
  </widget>
  <widget class="QPushButton" name="profileConfirm">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>151</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#profileConfirm{
background-image: url(:/uielements/uielements/okNoPaint.PNG) 0 0 0 0 stretch stretch;
background-color: rgb(0, 0, 0);
}

QPushButton#profileConfirm:hover{
	background-image: url(:/uielements/uielements/ok.PNG) 0 0 0 0 stretch stretch;

border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>170</y>
     <width>211</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#cancel{
background-image: url(:/uielements/uielements/cancelNoPaint.PNG) 0 0 0 0 stretch stretch;
background-color: rgb(0, 0, 0);
}

QPushButton#cancel:hover{
	background-image: url(:/uielements/uielements/cancel.PNG) 0 0 0 0 stretch stretch;

border: none;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>cancel</zorder>
  <zorder>profileConfirm</zorder>
  <zorder>profileLabel</zorder>
  <zorder>profileNameLineEdit</zorder>
  <zorder>profileModNumberSpinBox</zorder>
  <zorder>profileNumberLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
